src/Application/inc/Calculator.h

```diff
@@ -28,7 +28,7 @@ class Calculator
          * @brief initiates calculator and works with the parser to return a result object
          * @param [in] history a pointer to a created history class usd to track completed expressions
         */
-        void runCalculator(std::shared_ptr<calculator::History> &history);
+        void runCalculator(std::shared_ptr<calculator::History> const &history);
```

Comment:
Hmmm not sure what the const means when it is here.... it should be `const std::shared_ptr<calculator::History> &history`

Reply:
I guess it means the same thing .... ? https://stackoverflow.com/questions/5503352/const-before-or-const-after

Anyway... you should put the const before the type... specifically because `std::shared_ptr<calculator::History> & const history` does mean something different and it could be easy to mix them up